docker/Dockerfile.alpine

```diff
@@ -10,30 +10,35 @@ RUN apk add --update \
   flex-dev \
   git \
   linux-headers \
-  llvm5-dev \
-  llvm5-static \
+  llvm8-dev \
+  llvm8-static \
   python \
   zlib-dev
 
 # Put LLVM directories where CMake expects them to be
-RUN ln -s /usr/lib/cmake/llvm5 /usr/lib/cmake/llvm
-RUN ln -s /usr/include/llvm5/llvm /usr/include/llvm
-RUN ln -s /usr/include/llvm5/llvm-c /usr/include/llvm-c
+RUN ln -s /usr/lib/cmake/llvm8 /usr/lib/cmake/llvm
+RUN ln -s /usr/include/llvm8/llvm /usr/include/llvm
+RUN ln -s /usr/include/llvm8/llvm-c /usr/include/llvm-c
 
 # Alpine currently does not have a package for bcc. Until they do,
 # we'll peg the alpine build to bcc v0.8.0
 #
 # We're building here so docker can cache the build layer
 WORKDIR /
-RUN curl -L https://github.com/iovisor/bcc/archive/v0.8.0.tar.gz \
-  --output /bcc.tar.gz
-RUN tar xvf /bcc.tar.gz
-RUN mv bcc-0.8.0 bcc
+
+RUN git clone --depth 1 --branch v0.10.0 https://github.com/iovisor/bcc.git && \
```

Comment:
There's an Alpine [package](https://pkgs.alpinelinux.org/packages?name=bcc*&branch=edge) for bcc now that should let us remove this compile step and will resolve one of the sed hacks. Currently only available in `alpine:edge`, but we should see 3.10.3 fairly soon.

Reply:
Yes, I think we should switch to using alpine:edge if it means we don't have to recompile bcc in the CI. If edge is too buggy we can re-evaluate. I think half the CI time is spent building bcc.

Reply:
@acj  will there be a static build of bcc too?

Reply:
Yes, that's on my to-do list. I'll look into it this week.

Reply:
Yes, that's on my to-do list. I'll look into it this week.

Reply:
Ah nice. I'm not familiar with alpine, will it show up in edge as soon as you "release" the package or will that take a while?

Reply:
It will be available in edge as soon as the change is merged and the package index is rebuilt, which normally happens within a few hours. bcc is available in Alpine 3.11 as of ~2 weeks ago. I'm not sure whether the -static subpackage will make it into 3.11.x or be delayed until 3.12.

We'll know soon enough :)  https://gitlab.alpinelinux.org/alpine/aports/merge_requests/2654

My bpftrace changes are [here](https://github.com/iovisor/bpftrace/compare/master...acj:use-alpine-bcc-package). A handful of test failures (mostly codegen, not sure why; will dig into it later), but it seems to work.

Happy new year, everyone.

Reply:
[bcc-static](https://pkgs.alpinelinux.org/packages?name=bcc-static&branch=edge) is now available in edge. Working on adding it to 3.11 as well.

Reply:
> My bpftrace changes are here. A handful of test failures (mostly codegen, not sure why; will dig into it later), but it seems to work.

Might have more luck if you switch to llvm8.

Reply:
Ah, yes indeed. Using llvm8 creates a dependency conflict that's ugly to resolve (see my updated branch), but I can get the tests to pass locally. There's no clang8 package on 3.11, so I think we'll be dependent on the older clang package from 3.10 unless/until we update to llvm9.

It looks like we're not finding `LIBBCC_LOADER_LIBRARY_STATIC` when I run the vanilla CI build script. I'll look into that next.